packages/office-ui-fabric-react/src/components/Panel/Panel.scss

```diff
@@ -92,6 +92,7 @@ $CommandBarHeight: 40px;
     @include right(auto);
     @include left(0);
     width: $Panel-width-xs;
+    @include drop-shadow(30px, 0px, 30px, -30px, .2);
```

Comment:
Let's pull this in. Looks like core 7.0 will bring in the required mixing. We can update it then.